README.rst

```diff
@@ -1,5 +1,4 @@
-smock
-=====
+.. image:: https://user-images.githubusercontent.com/14298799/128897259-1d2c43b5-9156-425e-82e0-ab13f259e57c.gif
```

Comment:
Let's make the logo smaller

Reply:
Let's add it as well in the docs side bar, something like [waffle](https://ethereum-waffle.readthedocs.io/en/latest/)

Reply:
Let's add the **S** as a favicon in the docs